r/R/csv.R

```diff
@@ -61,8 +61,8 @@
 #' * "l": `bool()`
 #' * "f": `dictionary()`
 #' * "D": `date32()`
-#' * "T": `timestamp()`
-#' * "t": `time32()`
+#' * "T": `timestamp(unit = "ns")`
+#' * "t": `time32(unit = "ms")`
```

Comment:
```suggestion
#' * "t": `time32()`
```
Let's leave this as it is as this is the default value

Reply:
This is the default value in the current implementation. However, I think it should be explicit because it is not explicit for users looking at the help page for this function.

``` r
arrow::time32() == arrow::time32(unit = "ms")
#> [1] TRUE
arrow::time32() == arrow::time32(unit = "s")
#> [1] FALSE
```

<sup>Created on 2022-02-25 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Even if it should not be changed in this PullRequest, it should be changed.

Reply:
OK, fine to update it but add a comment after pointing out this is the default value, so there's no ambiguity.  And let's leave it as it is (without the default option explicitly stated) in the code.